src/Stock/Core/Database/DBConnection.php

```diff
@@ -11,6 +11,8 @@ class DBConnection
      */
     protected $db_config;
 
+    public $id;
```

Comment:
DBConnection nie potrzebuje propertisa $id.
Nawet go nigdznie nie wykorzystuje.
Gdybyśmy mieli wiele różnych połączeń z bazami danych, które chcieli byśmy z jakiegoś powodu rozrózniać po ID to wtedy by to miało sens, a tak to jest tylko nieużywany element.
Trzeba to usunąć.

Prawdopodobnie chciałeś to dodać do class Stock, ponieważ tam brakuje $id